docs: improve Community Section (#897)
iresty/iresty.github.io: website/docs/general/community.md

description: This article provides useful information about Apache APISIX's Community, including 2 ways to join the Slack channel.
---

Apache APISIX is a project maintained by community members under the Apache Software Foundation.

> It is an open source project that abides by the [Apache code of conduct](https://www.apache.org/foundation/policies/conduct.html). Apache APISIX is a dynamic, real-time, high-performance Cloud-Native API Gateway. It handles traditional north-south traffic, as well as east-west traffic between services.

Reviewer: ```suggestion
It is an open source project that abides by the [Apache Software Foundation's code of conduct](https://www.apache.org/foundation/policies/conduct.html). Apache APISIX is a dynamic, real-time, high-performance Cloud-Native API Gateway. It handles traditional north-south traffic, as well as east-west traffic between services.
```

Reviewer: Might as well just use the full name

Want to get involved with a project, join our slack community, collaborate & interact with other community members who will make sure you are not left out. We are open to any form of contribution; like code contributions, design, community management, advocacy, or you can just to listen in.

### Joining the Mailing List

### Get help from Mailing List (recommended)
The Apache APISIX project is built by the collaboration of the community and great contributors. Catch up on other activities in this community by subscribing to the [mailing list](https://apisix.apache.org/docs/general/subscribe-guide).

### Joining the Slack Channel

Apache APISIX provides 2 ways to join the Slack channel.

1. Subscribe the mailing list according to [this guide](subscribe-guide.md);
2. Send a mail to [dev@apisix.apache.org](mailto:dev@apisix.apache.org) and wait for reply.
1. You can join the slack channel using [this invite](https://join.slack.com/t/the-asf/shared_invite/zt-vlfbf7ch-HkbNHiU_uDlcH_RvaHv9gQ).

### Use the invitation link below
   _Please open an issue if this link is expired._

> Please use this link [https://join.slack.com/t/the-asf/shared_invite/zt-vlfbf7ch-HkbNHiU_uDlcH_RvaHv9gQ](https://join.slack.com/t/the-asf/shared_invite/zt-vlfbf7ch-HkbNHiU_uDlcH_RvaHv9gQ). If this link is expired, please use the mailing list way.

Reviewer: Nice work hiding the long URL as a hyperlink.

description: This article provides useful information about Apache APISIX's Community, including 2 ways to join the Slack channel.
---

Apache APISIX is a project maintained by community members under the Apache Software Foundation.
It is an open source project that abides by the [Apache Software Foundation's code of conduct](https://www.apache.org/foundation/policies/conduct.html). Apache APISIX is a dynamic, real-time, high-performance Cloud-Native API Gateway. It handles traditional north-south traffic, as well as east-west traffic between services.

> ## How to get involved

Reviewer 2: ```suggestion
## How to Get Involved
```

Since we are using title case on other headings throughout this docs.

> Apache APISIX is an open source infrastructure software project that provides API processing and analytics with products and solutions for microservices and real-time traffic processing; such as API gateway, k8s ingress controller, and Service Mesh.

Reviewer 2: Doesn't the first sentence in this doc already cover this? Feels like we are repeating this here again.

Author: this is a lot different from the first sentence. Its also a good transition into getting involved

Want to get involved with a project, join our slack community, collaborate & interact with other community members who will make sure you are not left out. We are open to any form of contribution; like code contributions, design, community management, advocacy, or you can just to listen in.

### Joining the Mailing List

> The Apache APISIX project is built by the collaboration of the community and great contributors. Catch up on other activities in this community by subscribing to the [mailing list](https://apisix.apache.org/docs/general/subscribe-guide).

Reviewer 2: ```suggestion
The Apache APISIX project is built by the collaboration of the community and great contributors. Catch up on other activities in this community by [subscribing to the mailing list](https://apisix.apache.org/docs/general/subscribe-guide).
```

### Joining the Slack Channel

> Apache APISIX provides 2 ways to join the Slack channel.

Reviewer 2: ```suggestion
You can join the Apache APISIX Slack channel in two ways:
```

> 1. You can join the slack channel using [this invite](https://join.slack.com/t/the-asf/shared_invite/zt-vlfbf7ch-HkbNHiU_uDlcH_RvaHv9gQ).
> 
>    _Please open an issue if this link is expired._
> 
> 2. You can also join via the mailing list by sending an email to the list ([dev@apisix.apache.org](mailto:dev@apisix.apache.org)) requesting help to join.

Reviewer 2: ```suggestion
- using [this invite](https://join.slack.com/t/the-asf/shared_invite/zt-vlfbf7ch-HkbNHiU_uDlcH_RvaHv9gQ).

   _Please open an issue if this link is expired._

- By sending an email to the list ([dev@apisix.apache.org](mailto:dev@apisix.apache.org)) requesting help to join.
```

description: This article provides useful information about Apache APISIX's Community, including 2 ways to join the Slack channel.
---

Apache APISIX is a project maintained by community members under the Apache Software Foundation.
It is an open source project that abides by the [Apache code of conduct](https://www.apache.org/foundation/policies/conduct.html). Apache APISIX is a dynamic, real-time, high-performance Cloud-Native API Gateway. It handles traditional north-south traffic, as well as east-west traffic between services.

## Apache APISIX Community

> Apache APISIX is an open source infrastructure software company that provides API processing and analytics with products and solutions for microservices and real-time traffic processing; such as API gateway, k8s ingress controller, and Service Mesh.

Reviewer 3: ```suggestion
Apache APISIX is an open source infrastructure software project that provides API processing and analytics with products and solutions for microservices and real-time traffic processing; such as API gateway, k8s ingress controller, and Service Mesh.
```

I think the project would be better. Since Apache APISIX is a project only.

> The Apache APISIX project is built by collaboration of the community and great contributors. Catch up on other activities in this community by subscribing to the [mailing list](dev-subscribe@apisix.apache.org).

Reviewer 4: ```suggestion
The Apache APISIX project is built by the collaboration of the community and great contributors. Catch up on other activities in this community by subscribing to the [mailing list](https://apisix.apache.org/docs/general/subscribe-guide).
```
